.dev/tests/phpunit/includes/test-coblocks-block-assets.php

```diff
@@ -99,6 +99,24 @@ public function test_editor_assets_scripts() {
 		$this->coblocks_block_assets->editor_assets();
 
 		$this->assertTrue( array_key_exists( 'coblocks-editor', $wp_scripts->registered ) );
+
+		$path  = dirname( __FILE__ ) . '/../../../../dist/coblocks-*.js';
+		$files = glob( $path );
+
+		foreach( $files as $file ) {
+			$dist_script = pathinfo( $file, PATHINFO_FILENAME );
+
+			if ( $dist_script == 'coblocks-plugin-deactivation' ) {
```

Comment:
Not sure if it was intentional to use a double `=` and not triple like you do below, but thought I would point it out :)